bigbluebutton/greenlight
# Setting to allow upstream auth user attributes overrides

## Description

Refs #2353

This introduces a setting to select which user attributes to take over from upstream authentication methods on user login.

## Testing Steps

- Create a Greenlight user through an omniauth authentication provider
- Change name in the authentication provider
- Re-login to Greenlight, the Greenlight user name should match the authentication provider user name.
